app/views/partials/_navbar.html.erb

```diff
@@ -11,7 +11,7 @@
         </ul>
         <ul class="navbar-nav ml-auto">
             <li class="nav-item">
-                <a class="nav-link" href="" data-target="#myModal" data-toggle="modal">Cart</a>
+                <a class="nav-link" href="">Cart</a>
```

Comment:
Please do not use the word Cart. It really sounds so 90-ties

Reply:
I love it, I'm a child of the 90's. But maybe your right and I need to refresh my views..